doc/api/errors.md

```diff
@@ -1312,6 +1312,14 @@ strict compliance with the API specification (which in some cases may accept
 `func(undefined)` and `func()` are treated identically, and the
 [`ERR_INVALID_ARG_TYPE`][] error code may be used instead.
 
+<a id="ERR_MISSING_DYNAMIC_INSTANTIATE_HOOK"></a>
+### ERR_MISSING_DYNAMIC_INSTANTIATE_HOOK
+
+> Stability: 1 - Experimental
+
+An [ES6 module][] loader hook specified `format: 'dynamic` but did not provide a
```

Comment:
`` `format: 'dynamic` `` — missing single quote.

Reply:
@vsemozhetbyt Thanks, fixed in #21492.